lib/full-site-editing/block-templates.php

```diff
@@ -137,6 +137,19 @@ function _gutenberg_add_template_part_area_info( $template_info ) {
 	return $template_info;
 }
 
+function get_inner_blocks_recursive( $blocks, $result = array() ) {
+	global $result;
```

Comment:
That's a bit weird, we can probably reuse the result of the recursive call and `array_merge` it